SPU-Horizon/gpa-frontend
# Getting rid of unused files within the repository.

This PR will get rid of un-used files on the front end.

To Test: 

Run the development environment and see if the application runs smoothly apart from these files.
